test/serverRendering_test.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import ReactDOMServer from 'react-dom/server';
+import { StyleSheetServer } from 'aphrodite';
+import { expect } from 'chai';
+import aphroditeInterface from 'react-with-styles-interface-aphrodite';
+
+import ThemedStyleSheet from '../src/ThemedStyleSheet';
+import { css, withStyles, withStylesPropTypes } from '../src/withStyles';
+
+describe('server rendering', () => {
+  const defaultTheme = {
+    color: {
+      red: '#990000',
+    },
+  };
+
+  ThemedStyleSheet.registerTheme(defaultTheme);
```

Comment:
these should be in a `beforeEach`

Reply:
Oops, took them out right before I committed.